config/locales/en.notifier.yml

```diff
@@ -16,7 +16,16 @@ en:
         outro: "All prices are estimates. Actual cost is assigned when the order is complete."
         total: Total
       order_status_changed_to_approved:
-        subject: "Reservation #%{order_detail} has been approved"
+        subject:
+          item: "Order #%{order_detail} has been approved"
+          bundle: "Reservation #%{order_detail} has been approved"
```

Comment:
I'm not sure this is accurate for bundles. What if the bundle is just services? I'm not even quite sure how this feature behaves with bundles. Can you set the default order status on a bundle? I thought each detail would take the default status of the individual product.